src/views/edition/components/sidebar.vue

```diff
@@ -79,18 +81,34 @@ export default Vue.extend({
         canCopy(): boolean {
             return this.newCopyName.trim().length > 0;
         },
-        current(): ScrollVersionInfo {
-            return this.$store.state.scroll.scrollVersion;
+        current(): EditionInfo {
+            return this.$store.state.edition.edition;
```

Comment:
Maybe `this.$store.state.edition.edition` should become `this.$store.state.edition.editionId` or `this.$store.state.edition.id`.  Pretty sure this was my fault.

Reply:
`this.$store.state` contains the application wide state. `$this.store.state.edition` contains edition-relevant state. `$this.store.state.edition.edition` contains the current edition - not an id. I think we'll rename it to `$this.store.state.edition.current`. It will still contain an edition, but it will be very clear it's the current edition. You can then use `$this.store.state.edition.current.id` if you want.